prow/config.yaml

```diff
@@ -37,5 +38,19 @@ presubmits: # runs on PRs
       containers:
       - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.1 # created by running `docker build -t <image> .` in the integration-job directory.
 
+postsubmits:
+  kyma-project/kyma:
+  - name: kyma-integration
+    branches:
+    - master
+    context: kyma-integration
+    skip_report: false # from documentation: SkipReport skips commenting and setting status on GitHub.
```

Comment:
```suggestion
    skip_report: true # from documentation: SkipReport skips commenting and setting status on GitHub.
    optional: true
```

Reply:
There is no `optional` and `skip_report` configurations on postsubmits jobs therefor I have removed it.